subsys/mpsl/init/mpsl_init.c

```diff
@@ -324,6 +430,29 @@ static void mpsl_calibration_work_handler(struct k_work *work)
 }
 #endif /* CONFIG_CLOCK_CONTROL_NRF_K32SRC_RC && !CONFIG_SOC_SERIES_NRF54HX */
 
+#if IS_ENABLED(CONFIG_MPSL_USE_ZEPHYR_PM)
+static void mpsl_pm_work_handler(struct k_work *work)
+{
+	ARG_UNUSED(work);
+	m_pm_work();
+}
+
+static void m_pm_init(void)
+{
+	mpsl_pm_params_t params = {0};
+	/* So that PM doesn't shut us off power when we are idle.*/
+	/* see min. residency times: https://github.com/nrfconnect/sdk-sysctrl/blob/master/sysctrl/conf/pm.overlay#L19*/
+	/* we want max sleep?? -> PM_MAX_LATENCY_IDLE_US < 5000000us*/
```

Comment:
Maybe this is not needed and should rather be determined by the max latency for starting to process an HCI command? We will not go to system off automatically - that has to be done explicitly by the application.